src/index.html

```diff
@@ -9,10 +13,40 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="../src/styles/blocks/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card"
+     data-qa="card">
```

Comment:
The `<div>` tag is incorrectly self-closed on line 20. In HTML, `<div>` elements should not be self-closing. You should remove the self-closing slash to ensure the HTML is valid.